pages/_app.tsx

```diff
@@ -1,6 +1,6 @@
 import Header from "../components/Header";
```

Comment:
Slipped by on a previous PR
```suggestion
import Header from "@/components/Header";
```